Gemfile.lock

```diff
@@ -23,6 +23,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
I think this is caused by my loading bundler on Windows. Probably adding .gitignore and ignore this file should suffice.